netbeans.apache.org/src/content/kb/docs/ide/git.asciidoc

```diff
@@ -35,8 +35,6 @@ Git is a free and open source, distributed version control system designed to ha
 
 === Contents
```

Comment:
Hi @mcdonnell-john 
The website already includes a "Table of Contents" for each page in the tutorials (and elsewhere), so we need to remove the "Contents" section and the manual list of headings below. This will be automatically generated.